04-reporting.Rmd

```diff
@@ -510,17 +509,176 @@ kableExtra: https://cran.r-project.org/web/packages/kableExtra/index.html
 
 
 
-## Results from statistical tests
+## Statistical Models and Tests
+
+When creating fully reproducible reports of empirical studies, it is obviously necessary to have
+a means to include the results from statistical models and tests.
+However, what comes out of R cannot be included in a report easily.
+Consider the output from a simple *t* test:
+
+```{r}
+t.test(yield ~ N, data = npk)
+```
+
+We created the function `apa_print()` to format the results from various statistical
+methods in accordance with APA guidelines.
+Just drop the results from these methods into `apa_print()`, and it will return
+a simple list with all you need to report the analysis.
+
+```{r}
+apa_print(
+  t.test(yield ~ N, data = npk)
+)
+```
+
+The returned list contains four elements:
+
+- `estimate`: A character string that holds the information about original-scale and/or standardized estimates of the model term.
+- `statistic`: A character string that holds the result of the NHST with test statistic, *df*s, and *p* value;
+- `full_result`: combines this information.
+- `table`: arranges model terms, estimates and test statistics in a `data.frame`
+
+List elements `$estimate`, `$statistic`, and `$full_result` are intended to be
+included directly into the body of your manuscript by using inline code chunks.
+For instance, you could first save the result from `apa_print()` in an output object
+
+```{r}
+out <- apa_print(
+  t.test(yield ~ N, data = npk)
+)
+```
+
+and then include a single list element by writing:
+
+```{r inline-results, eval = FALSE, highlight = FALSE}
```

Comment:
Wouldn't it make sense to just turn this into a regular code block (i.e. wrapped in just three backticks or three tildes)?

Reply:
Copied this from an earlier version. I just tried what you suggested, but then the R inline chunk is evaluated in place